AWScliSetup.sh

```diff
@@ -105,7 +106,7 @@ enable_services()
 
 get_awstools_filenames()
 {
-   if [[ -z ${AWSTOOLSRPM} ]]
+   if [[ -z ${AWSTOOLSRPM:-} ]]
```

Comment:
Why set the default in two places?

Reply:
Meant to only set it once and forgot that I'd done a null-declare at the top